sql/core/benchmarks/DataSourceReadBenchmark-jdk11-results.txt

```diff
@@ -2,322 +2,430 @@
+SQL Parquet Vectorized: DataPageV2 (Nested Column Enabled)             150            174          27        104.7           9.6      14.6X
+
+OpenJDK 64-Bit Server VM 11.0.14+9-LTS on Linux 5.13.0-1021-azure
+Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+SQL Single SMALLINT Column Scan in Struct:                   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-------------------------------------------------------------------------------------------------------------------------------------------
+SQL ORC MR                                                            2322           2391          97          6.8         147.6       1.0X
+SQL ORC Vectorized (Nested Column Disabled)                           2362           2374          17          6.7         150.2       1.0X
+SQL ORC Vectorized (Nested Column Enabled)                             412            419           9         38.2          26.2       5.6X
+SQL Parquet MR: DataPageV1                                            2393           2400          10          6.6         152.1       1.0X
+SQL Parquet Vectorized: DataPageV1 (Nested Column Disabled)           2919           2922           4          5.4         185.6       0.8X
+SQL Parquet Vectorized: DataPageV1 (Nested Column Enabled)             228            281          54         69.0          14.5      10.2X
+SQL Parquet MR: DataPageV2                                            2223           2240          25          7.1         141.3       1.0X
+SQL Parquet Vectorized: DataPageV2 (Nested Column Disabled)           2692           2712          28          5.8         171.2       0.9X
+SQL Parquet Vectorized: DataPageV2 (Nested Column Enabled)             341            361          31         46.1          21.7       6.8X
+
+OpenJDK 64-Bit Server VM 11.0.14+9-LTS on Linux 5.13.0-1021-azure
+Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+SQL Single INT Column Scan in Struct:                        Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-------------------------------------------------------------------------------------------------------------------------------------------
+SQL ORC MR                                                            2376           2380           6          6.6         151.0       1.0X
+SQL ORC Vectorized (Nested Column Disabled)                           2333           2378          64          6.7         148.4       1.0X
+SQL ORC Vectorized (Nested Column Enabled)                             430            451          20         36.6          27.3       5.5X
+SQL Parquet MR: DataPageV1                                            2485           2501          22          6.3         158.0       1.0X
+SQL Parquet Vectorized: DataPageV1 (Nested Column Disabled)           3017           3062          65          5.2         191.8       0.8X
```

Comment:
Do you know the reason why this is slower than all ORC readers by at least 20%?

Reply:
I'm not sure. This part is calling the row-based Parquet reader from `parquet-mr` which I'm not very familiar with. Overall its performance is not very good.